docs/usage.md

```diff
@@ -0,0 +1,72 @@
+# Usages
+Modelina can be used in many different contexts and has many features all depending on the output language, this document will show you the very basic usages of the library. 
```

Comment:
Modelina can be used in many different contexts and has many features, all depending on the output language. This document will walk you through you the library's basic usages.